Directory.Build.props

```diff
@@ -7,6 +7,12 @@
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <BuildArchitecture>$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant())</BuildArchitecture>
+    <Architecture Condition="'$(BuildArchitecture)' == 'arm64'">$(BuildArchitecture)</Architecture>
```

Comment:
My first instinct was to default to `$(BuildArchitecture)` always, but that breaks VS (x86) on Windows. See https://github.com/dotnet/corefx/pull/40424 for more information.